examples/webapp/main.go

```diff
@@ -135,6 +140,9 @@ func help(w http.ResponseWriter, r *http.Request) {
 	helpTxt += "Sign in of existing user with magiclink sms go to: /magiclink/signin?sms=\n\n"
 	helpTxt += "Sign in of existing user with magiclink whatsapp go to: /magiclink/signin?whatsapp=\n\n"
 	helpTxt += "---------------------------------------------------------\n\n"
+	helpTxt += "Sign up with enchantedlink go to: /enchantedlink/signup?email=\n\n"
```

Comment:
Change to "enchanted link", when they are connected it is hard to read.

Reply:
done